pkg/operator/controller/status_test.go

```diff
@@ -39,97 +36,98 @@ func TestComputeOperatorStatusConditions(t *testing.T) {
 		},
 		{
 			description: "0/0 dns resources available",
+			dnses:       dnsStatusConditionsCounts{0, 0, 0, 0},
 			expected:    conditions{true, true, false},
 		},
 		{
 			description: "1/2 dns resources available",
-			dnses:       dnsesCount{1, 2},
+			dnses:       dnsStatusConditionsCounts{1, 1, 1, 2},
 			expected:    conditions{true, true, true},
```

Comment:
Could we use field labels in these structs to visually hint at the value meanings?

Reply:
Yeah, at one point we tried to make them more compact, but I can expand them again.

Reply:
Fixed.  Does it look all right?